include/rmlab/Notebook.cpp

```diff
@@ -75,23 +76,31 @@ namespace detail
         // select 3:   0 (brush)
         // select 4:   5 (marker/highlighter: always color 0)
         // what is/was 6? :-)
+        std::cout << "brush type" << "\n";
         fstream.read( (char*)&curLine.brush_type, sizeof(int32_t) );
 
+        
+        std::cout << "color" << "\n";
         // color (0: black, 1: grey, 2: white)
         fstream.read( (char*)&curLine.color, sizeof(int32_t) );
 
+        std::cout << "line attr" << "\n";
         // unknown 4 Byte (int32_t?), always zero?
         // non-stored information about "selected" lines?
         fstream.read( (char*)&curLine.unknown_line_attribute, sizeof(int32_t) );
 
+
+        std::cout << "brush size" << "\n";
         // brush base size: 1.875, 2.0, 2.125
         fstream.read( (char*)&curLine.brush_base_size, sizeof(float) );
+        std::cout << "npoints" << "\n";
         fstream.read( (char*)&curLine.npoints, sizeof(int32_t) );
 
-        for( int n = 0; n < curLine.npoints; ++n )
-        {
-            readPoint( fstream, curLine );
-        }
+        // for( int n = 0; n < curLine.npoints; ++n )
```

Comment:
Can you please remove the dead code if its not needed anymore?
At which point do we read the actual point data now?